fili-core/src/test/groovy/com/yahoo/bard/webservice/web/FiliResponseWriterSpec.groovy

```diff
@@ -0,0 +1,167 @@
+// Copyright 2017 Oath Inc.
+// Licensed under the terms of the Apache license. Please see LICENSE.md file distributed with this work for terms.
+package com.yahoo.bard.webservice.web
+
+import static com.yahoo.bard.webservice.config.BardFeatureFlag.PARTIAL_DATA
+
+import com.yahoo.bard.webservice.data.dimension.Dimension
+import com.yahoo.bard.webservice.data.dimension.DimensionField
+import com.yahoo.bard.webservice.util.GroovyTestUtils
+import com.yahoo.bard.webservice.util.SimplifiedIntervalList
+
+import org.joda.time.Interval
+
+import spock.lang.Unroll
+
+class FiliResponseWriterSpec extends ResponseWriterSpec {
+
+    @Unroll
+    def "test for requested numeric metrics in the API response with the #linkNames links"() {
```

Comment:
This should probably be put on the spec for the Json and JsonApi specifications, since the whole metadata block only applies to the json/jsonapi mappers.

/NotABlocker